homeassistant/const.py

```diff
@@ -360,6 +360,7 @@
 HTTP_HEADER_CACHE_CONTROL = 'Cache-Control'
 HTTP_HEADER_EXPIRES = 'Expires'
 HTTP_HEADER_ORIGIN = 'Origin'
+HTTP_HEADER_X_FORWARDED_FOR = 'X-Forwarded-For'
```

Comment:
I wish I could get rid of this change.  I reset any change in my branch by `git checkout upstream/master homeassistant/const.py` and it resolved the above conflict but wont remove this.  Not sure where the change came from :(